### DISCUSSION TITLE
Vercel build is failing even if is saying it's completed

### DISCUSSION THREAD

**Discussion by Syesy** [vercel/vercel#8705] [Help]
After introducing Prismic, our builds have started to fail even if on Vercel's log there is a message saying that the build completes as you can see in this picture

![image](https://user-images.githubusercontent.com/17948756/194885082-8474340c-c66a-4f6d-bc25-f1f2c6b8a5ff.png)

On localhost when I run `vercel build` with the same configuration that is present on the remote vercel environment, the build completes without any issue

We're using
NodeJS version: 14.x
NextJS version: 12.0.9

The deployment url is: https://vercel.com/cesarine/frontend-main-github-integration/7fFrVWMaPDAEvSXaJFZpFQLCJabm

Do you have any clue on how to resolve this issue? Thanks!

**Comment by amyegan** [comment]
Hi @Syesy! It looks like a couple of members of your team have already opened support cases related to this problem. If you can provide a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will help the team get a better look at what is going wrong. Feel free to follow up here or in one of your teammates' cases, whichever is most comfortable for your team!

**Reply by Syesy** [reply]
Thanks @amyegan I've replied by email
